### ISSUE TITLE
Feature request: option to disable Italic font in syntax highlighter

### ISSUE THREAD

**Issue by karabaja4** [tsujan/FeatherPad#417]
Hello,

would it be possible to add an option to disable the italic style in the highlighter?

I find it it makes it harder for me read my code, I constantly lose focus when reading text that is switching between italic and non-italic style.

Thanks!

Igor

**Comment by tsujan** [comment]
Hi,

> would it be possible to add an option to disable the italic style in the highlighter?

It's possible but I'm afraid I can't do it. The number of font colors that can be clearly distinguished from each other by most users *at first glance* — which is important with a good syntax highlighting — is very limited. I've added italicity and boldness so that different syntax elements can be distinguished from each other easily. If I remove one of them, highlighting will become confusing.

In short, there's no coding obstacle; the problem is related to human eyes.
